public/data/AbhijithGanesh.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name":"Abhijith Ganesh",
+    "bio":"An Aspiring Tech afficionado and a freshman in computer science",
+    "avatar":"https://github.com/AbhijithGanesh.png",
+    "links":[
+        {
+            "name":"Follow me on GitHub",
+            "url":"https://github.com/AbhijithGanesh",
+            "icon":"github"
+        },
+        {
+            "name":"Conenct with me on LinkedIn",
+            "url":"https://linkedin.com/in/AbhijithGanesh14",
+            "icon":"linkedin"
+        },
+        {
+            "name":"Check my website out",

```

Comment:
you can write " Check Out my website " That will look more proper ( just a suggestion )